lib/DatedInterface.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Timber;
+
+/**
+ * Interface DateTimeInterface
+ *
+ * An interface for classes that implement getting date/time values from objects.
+ *
+ * @since 2.0.0
+ */
+interface DatedInterface {
```

Comment:
Maybe name it HasDate?

Reply:
I agree with @jarednova, interface names should be explicit about the fact that they're interfaces. `HasDate` sounds more like a trait to me.

Reply:
Let’s keep `DatedInterface` then 👍.